lib/cc/engine/analyzers/php/main.rb

```diff
@@ -20,7 +20,7 @@ class Main < CC::Engine::Analyzers::Base
           private
 
           def process_file(path)
-            code = File.read(path)
+            code = File.binread(path)
```

Comment:
I was debugging and changed it just to see if this had anything to do with it, it's something we should probably be doing anyways since the other analyzers are using `binread` anyways.